cmd/runtimetest/main.go

```diff
@@ -394,7 +450,7 @@ func validate(context *cli.Context) error {
 
 	linuxValidations := []validation{
 		validateDefaultFS,
-		validateDefaultDevices,
```

Comment:
On 09/14/2016 01:38 AM, W. Trevor King wrote:
> We want to keep this (as long as the spec requires runtimes to supply those devices).

validateDefaultDevices is not removed.
I have just merged it into validateLinuxDevices